src/registry.hpp

```diff
@@ -66,8 +66,8 @@ struct collection_of<Tag, std::tuple<U...>> {
 class registry_t::inner_t {
 public:
     collection_of<tag::gauge, std::tuple<std::int64_t, std::uint64_t, double>> gauges;
-    collection_of<tag::counter, std::tuple<std::int64_t, std::uint64_t>> counters;
-    collection_of<tag::meter, std::tuple<meter_t>> meters;
+    collection_of<tag::count, std::tuple<std::int64_t, std::uint64_t>> counters;
```

Comment:
Can you describe collection_of magic in comments. It's rather hard to understand.